docs/driver-parameters.md

```diff
@@ -47,6 +47,7 @@ protocol | file share protocol | `smb`, `nfs` | No | `smb`
 networkEndpointType | specify network endpoint type for the storage account created by driver. If `privateEndpoint` is specified, a private endpoint will be created for the storage account. For other cases, a service endpoint will be created for `nfs` protocol by default. | "",`privateEndpoint` | No | `` <br>for AKS cluster, make sure cluster Control plane identity (that is, your AKS cluster name) is added to the Contributor role in the resource group hosting the VNet
 location | specify Azure storage account location | `eastus`, `westus`, etc. | No | if empty, driver will use the same location name as current k8s cluster
 resourceGroup | specify the resource group in which Azure file share will be created | existing resource group name | No | if empty, driver will use the same resource group name as current k8s cluster
+clientID | specify Azure client ID who will create Azure file share | Azure client ID | No | if empty, kubelet MID will be used
```

Comment:
`kubelet MID` --> `kubelet managed identity`

and move this to L73 since this param is only for SMB protocol